gateway/gateway.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/gin-contrib/location"
 	"github.com/gin-contrib/static"
 	"github.com/gin-gonic/gin"
-	logger "github.com/ipfs/go-log"
+	logger "github.com/ipfs/go-log/v2"
```

Comment:
Just to make all packages in this repo use `v2` and avoid having two major versions.